build.gradle

```
@@ -1251,3 +1258,7 @@ tasks.withType(JavaExec).configureEach {
 task printVersion {
     println project.version
 }
+
+tasks.withType(Zip).configureEach {
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+}
```

```suggestion
}

```

should place last line feed.